docs/shared_parsers_catalog/system_time.rst

```diff
@@ -1,3 +1,38 @@
-.. automodule:: insights.parsers.system_time
+System time configuration
+=========================
+
+file ``/etc/chrony.conf``
+-------------------------
+.. autoclass:: insights.parsers.system_time.ChronyConf
+   :members:
+   :show-inheritance:
+
+file ``/etc/sysconfig/chronyd``
+-------------------------------
+.. autoclass:: insights.parsers.system_time.ChronydService
+   :members:
+   :show-inheritance:
+
+command ``file -L /etc/localtime``
+----------------------------------
+.. autoclass:: insights.parsers.system_time.LocalTime
+   :members:
+   :show-inheritance:
+
+file ``/etc/ntp.conf``
+----------------------
+.. autoclass:: insights.parsers.system_time.NTP_conf
+   :members:
+   :show-inheritance:
+
+command ``ntptime``
+-------------------
+.. autoclass:: insights.parsers.system_time.NtpTime
+   :members:
+   :show-inheritance:
+
+file ``/etc/sysconfig/ntpd``
+----------------------------
+.. autoclass:: insights.parsers.system_time.NTPDService
```

Comment:
@PaulWay the `NTPConfParser` class is missing from the documentation in this PR and I just noticed it was also missing from #109 .   Please update this PR and create a new PR in include this class in the 1.x documentation.